intermediate/indexing/advanced-indexing.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>Could also link to <a href="https://jakevdp.github.io/PythonDataScienceHandbook/02.07-fancy-indexing.html" target="_blank">https://jakevdp.github.io/PythonDataScienceHandbook/02.07-fancy-indexing.html</a></p><p> </p><p>and stick it in a <code>seealso</code> block </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Thanks for suggesting this. I added this and NumPy advanced indexing (https://numpy.org/doc/stable/user/basics.indexing.html#advanced-indexing) in the seealso block.